### PR TITLE
aws-workmail: add page

### PR DESCRIPTION
**Pull request by ShivamJoker**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): 2.8.2**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Comment by ShivamJoker** [comment]
@navarroaxel can you also review and approve it?

**Comment by kbdharun** [comment]
> navarroaxel can you also review and approve it?

I will request his review, If possible can you add Hindi translation to [subcommand-mention](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/subcommand-mention.md)?

> Thanks in advance

**Comment by ShivamJoker** [comment]
Sure thing.

**Comment by ShivamJoker** [comment]
@kbdharun what will be the format of commit message?

**Comment by kbdharun** [comment]
> @kbdharun what will be the format of commit message?

subcommand-mention: add Hindi translation

File: pages/common/aws-workmail.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# aws workmail
+
+> All Amazon WorkMail API operations using the CLI.
```
Comment (navarroaxel) [inline pages/common/aws-workmail.md:3]:
```suggestion
> Manage Amazon WorkMail using the CLI.
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**